### PR TITLE
Fixed skill delete refresh

### PR DESCRIPTION
**Pull request by sjain93**
setState correctly for deletion of skills from User UI
Still working on passing new state from child to parent to disregard refresh

### CODE REVIEW

File: client/src/pages/Profile.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ function Profile(props) {
   }
 
   const handleDelete = skill => () => {
-    console.log(skill.name);
+    // console.log(skill.name);
```
Comment (skassam21) [inline client/src/pages/Profile.js:120]: remove comment

File: server/resources/User.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def put(cls, user_id: int):
         if user and user.id == current_user:
             try:
                 #loading in form data through validation/clean by marshmallow
-                user.username = user_json["username"]
+                # user.username = user_json["username"]
```
Comment (skassam21) [inline server/resources/User.py:75]: was this supposed to be deleted? if so remove comment

**Review by skassam21 — APPROVED** [review]
makes sense!

File: server/resources/User.py (hunk 2 above)
Comment (sjain93) [reply server/resources/User.py:75]: Future state of the page should allow the user to edit their username, can remove from this iteration

**Pull request merged**